<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fight</class>
 <widget class="QWidget" name="fight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="fight_verticalLayout" stretch="1,0,0">
     <item>
      <layout class="QHBoxLayout" name="grid_horizontalLayout">
       <item>
        <widget class="QGroupBox" name="you_groupBox">
         <property name="title">
          <string>You:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="Grid" name="you_grid" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="opponent_groupBox">
         <property name="title">
          <string>Opponent:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="Grid" name="opponent_grid" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="fire_horizontalLayout">
       <item>
        <widget class="QLabel" name="coord_label">
         <property name="text">
          <string>Choose Coordinant:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="col_comboBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="row_comboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>FIRE!</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="text_horizontalLayout">
       <item>
        <widget class="QLabel" name="turn_label">
         <property name="text">
          <string>Turn:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="playerTurn_label">
         <property name="text">
          <string>Player #</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="statusUpdate_label">
         <property name="text">
          <string>hit/miss</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Grid</class>
   <extends>QWidget</extends>
   <header>grid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
